projects/plugins/boost/app/assets/src/js/react-components/pages/settings/elements/MinifyMeta.tsx

```diff
@@ -0,0 +1,88 @@
+import { useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { useState } from 'react';
+import { z } from 'zod';
+import { __, sprintf } from '@wordpress/i18n';
+
+export const minifyMetaTypes = {
+	minify_js_excludes: z.array( z.string() ),
+	minify_css_excludes: z.array( z.string() ),
+};
+
+type MinifyMetaSettings = keyof typeof minifyMetaTypes;
+
+interface Props {
+	type: MinifyMetaSettings;
+	inputLabel: string;
+	buttonText: string;
+	placeholder: string;
+	value: string[];
+}
+
+let nextIdIndex = 0;
+
+const MetaComponent = ( { inputLabel, buttonText, placeholder, type }: Props ) => {
+	const { data, mutate } = useDataSync( 'jetpack_boost_ds', type, minifyMetaTypes[ type ] );
```

Comment:
Can we call it something other than `type`? In this context where I don't know what the parameters of `useDataSync` are, `type` is a bit misleading. Maybe something like `option` or `optionKey`?

Reply:
Thanks for this - I also had a hard time understanding what it is and had to go digging 😅 .

Updated `type` to `optionKey`.